route-rs-runtime/src/processor/file_log.rs

```diff
@@ -0,0 +1,78 @@
+use crate::processor::{Log, Processor};
+use std::fmt::Debug;
+use std::fs::File;
+use std::marker::PhantomData;
+
+/// Wraps Log processor with a simpler File-specific interface
+/// You must provide a unique log filename.
+pub struct FileLog<A> {
+    phantom: PhantomData<A>,
+    log_processor: Log<A, File>,
+}
+
+impl<A> FileLog<A> {
+    pub fn new(name: &str) -> std::io::Result<FileLog<A>> {
```

Comment:
Good job using appropriate result here.  I guess Rust won't let you not get the types correct, but still, good stuff.